src/components/loader-bar/loader-bar.component.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { MarginProps } from "styled-system";
+import tagComponent from "../../__internal__/utils/helpers/tags/tags";
+import StyledLoaderBar, {
+  InnerBar,
+  StyledLoader,
+  StyledLoaderBarProps,
+} from "./loader-bar.style";
+
+export interface LoaderBarProps extends StyledLoaderBarProps, MarginProps {}
+
+const LoaderBar = ({
+  size = "medium",
+  ...rest
+}: LoaderBarProps): JSX.Element => {
```

Comment:
It's not really necessary to state that `JSX.Element` is returned from a component as usually its inferred automatically by TS

Reply:
I'll get it removed.